tools/esp8266/hmInverter.h

```diff
@@ -109,7 +109,7 @@ class Inverter {
         uint16_t      alarmMesIndex;         // Last recorded Alarm Message Index
         uint16_t      fwVersion;             // Firmware Version from Info Command Request
         uint16_t      powerLimit[2];         // limit power output
-        uint16_t      actPowerLimit;         //
+        float         actPowerLimit;         // actual power limit
```

Comment:
Why is this a float now, I thought we used fixed point integers ? Wouldn't it be better to do the typecast explicitly when we convert it to String somewhere ?

Reply:
Bei den ganzen anderen Werten(FLDs) wird leider auch alles zu einem float gemacht und später wieder zur Anzeige gerundet. Ich hasse floats wegen ihren Ungenauigkeiten, aber selbst double ist nicht perfekt :D
Man müsste die Rohdaten ebenfalls in uint16_t Speichern und bei der Anzeige die Kommastellen aus hmDefines holen. Das ist aber ein größerer Change für die Zukunft, so hat das PL nun wenigstens auch mal seine Kommastelle wieder, die ihm bisher geklaut wurde.